instrumentation/src/main/java/io/opentelemetry/android/features/diskbuffering/scheduler/DefaultExportScheduler.kt

```diff
@@ -14,11 +18,20 @@ class DefaultExportScheduler : PeriodicRunnable() {
     }
 
     override fun onRun() {
-        // TODO for next PR.
+        val exporter = SignalDiskExporter.get() ?: return
```

Comment:
Can we just pass an instance to this class instead of reaching out to a global static singleton?

Reply:
Yeah I think we could, though there are still benefits of having it as a singleton which I mentioned [here](https://github.com/open-telemetry/opentelemetry-android/pull/221#discussion_r1457393546). I think we should address that comment first before deciding how to handle this scenario.